apps/statistics/flow-query-certificates-pdf-conf.js

```diff
@@ -0,0 +1,20 @@
+'use strict';
+
+var db                = require('../../db')
+  , assign            = require('es5-ext/object/assign')
+  , dateToParamConfig = require('../../apps-common/query-conf/date-to');
+
+module.exports = [
+	require('../../apps-common/query-conf/date-from'),
+	assign({},
+		dateToParamConfig,
+		{
+			ensure: function (value) {
+				if (!value) return new db.Date();
+				return dateToParamConfig.ensure(value);
```

Comment:
Yes, with `service` and `certificate` it's true, in that case we need to pass it to server, understood.
Still then probably we can build this conf on spot in controller, as we do in other cases where it's not reusable.

With `dateTo` I don't see any difference, so I think customizations is unjustified.